drake/systems/plants/collision/BulletModel.cpp

```diff
@@ -299,6 +299,40 @@ namespace DrakeCollision
     return c.getResults();
   }
 
+  bool BulletModel::collidingPointsCheckOnly(const vector<Vector3d>& points,
+                                             double collision_threshold)
+  {
+    // Create sphere geometry
+    btSphereShape bt_shape(collision_threshold);
+
+    // Create Bullet collision object
+    btCollisionObject bt_obj;
+    btCollisionObject* bt_obj_ptr = static_cast<btCollisionObject*>(&bt_obj);
```

Comment:
@avalenzu I just copied and changed the collidingPoints function to return a boolean, so I didn't notice that. I think you are the author of that function :) Should I investigate that further?